semgrep_agent/semgrep_app.py

```diff
@@ -45,6 +47,29 @@ def report_start(self) -> None:
         else:
             self.scan_id = response.json()["scan"]["id"]
 
+    def fetch_rules_text(self) -> str:
+        """Get a YAML string with the configured semgrep rules in it."""
+        if self.scan_id is None:
+            raise RuntimeError("sapp is configured so we should've gotten a scan_id")
```

Comment:
sapp?

Reply:
`Sapp` is the client class and it's at `ctx.obj.sapp`. This is just here to appease mypy, so I thought I'd use the dev-facing name. I could rename it to `semgrep_app` everywhere?

Reply:
nah it's cool, I was just confused